### ISSUE TITLE
Add Live Preview explanation to /from-wordpress-to-statamic

### ISSUE THREAD

**Issue by Technobabble17** [statamic/docs#1505]
The live preview capability is a big feature worth mentioning in the Gutenberg section. Clients and content managers have grown to like the instant design feedback, and review that Gutenberg provides.

Providing a .gif of a text update would really excite those non devs/content managers.

**Comment by jackmcdade** [comment]
Does Gutenberg have live preview? Is this one of those "win-on-the-statamic-side" things?

**Comment by Technobabble17** [comment]
No, Wordpress does something quite annoying for developers... The page builder lets you add react blocks that let you modify text/images/settings directly on the block/page builder. So you are, in a way, live editing the page. It's a nightmare to create and maintain custom blocks, especially for non react devs.
